framework/src/engine/endpoint/txpool.ts

```diff
@@ -108,15 +108,16 @@ export class TxpoolEndpoint {
 		const header = this._chain.lastBlock.header.toObject();
 
 		if (!req.skipVerify) {
-			const { result } = await this._abi.verifyTransaction({
+			const { result, errorMessage } = await this._abi.verifyTransaction({
 				contextID: Buffer.alloc(0),
 				transaction: transaction.toObject(),
 				header,
 			});
 			if (result === TransactionVerifyResult.INVALID) {
 				return {
-					success: false,
+					result: TransactionVerifyResult.INVALID,
 					events: [],
+					errorMessage,
```

Comment:
Also, in [above](https://github.com/LiskHQ/lisk-sdk/pull/7817/files#diff-0cd3fada1cf6e016bb26ff686cebe60a25c8e2f04def5494db36df6ba7140356R65) postTransaction, we can replace error message